<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>generalconfig</class>
 <widget class="QWidget" name="generalconfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>208</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>505</width>
    <height>208</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>187</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_file">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>File</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_textColor">
        <property name="text">
         <string>Default Color for Text :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="KColorCombo" name="textColor"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_authorColor">
        <property name="text">
         <string>Default Color for Author Info :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="KColorCombo" name="authorColor">
        <property name="color">
         <color>
          <red>4</red>
          <green>255</green>
          <blue>96</blue>
         </color>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rdb_autoSource">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Auto Source :</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="rdb_manualSource">
        <property name="text">
         <string>Manual Source :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KUrlRequester" name="manualSource">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="autoSource">
        <property name="text">
         <string>Select Auto Source</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KColorCombo</class>
   <extends>QComboBox</extends>
   <header>kcolorcombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
